pkgs/development/python-modules/opentelemetry-proto/default.nix

```diff
@@ -18,7 +18,7 @@ buildPythonPackage {
 
   build-system = [ hatchling ];
 
-  dependencies = [ protobuf ];
+  dependencies = [ protobuf4 ];
```

Comment:
Please don't propagate such a versioned package.

> Only unversioned attributes (e.g. pydantic, but not pypdantic_1) can be included in dependencies, since due to PYTHONPATH limitations we can only ever support a single version for libraries without running into duplicate module name conflicts.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
You're right, I've fixed it in the meantime.